### PR TITLE
Make scippbuildtools work with other repos

### PR DESCRIPTION
**Pull request by nvaytet**

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,22 @@
 package:
   name: scippbuildtools
-  version: {{ GIT_DESCRIBE_TAG }}
+  version: 0.0.1
```
Comment (OwenArnold) [inline conda/meta.yaml]: Why use hard-coded versions rather than tags?

Comment (nvaytet) [reply conda/meta.yaml]: Because until there is a release made, the git tag variables don't seem to work.

Comment (OwenArnold) [reply conda/meta.yaml]: You would just need to create a 0.0.1 tag from main.

Comment (nvaytet) [reply conda/meta.yaml]: Yes that's the plan, but I'll do it once I've finalized everything

File: .azure_pipelines/main.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  type: boolean
+  default: false
+
+resources:
+  repositories:
+  - repository: SharedPipelines
+    name: scipp/pipelines
+    type: github
+    endpoint: scipp
+    ref: add_build_docs_option
```
Comment (OwenArnold) [inline .azure_pipelines/main.yml]: Upstream feature branch now merged into main

**Review by OwenArnold — CHANGES_REQUESTED** [review]
Basically looks good.

Could you extend the README on the repo to better explain what this is doing and how it fits into the ecosystem?

**Pull request merged**